pkgs/by-name/kn/knossosnet/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  buildDotnetModule
+, fetchFromGitHub
+, fontconfig
+, lib
+, openal
+, stdenv
+, xorg
+}:
+
+let
+  version = "0.2.0-RC10";
+  executable = "Knossos.NET";
+in
+buildDotnetModule {
+  pname = "knossosnet";
+  inherit version;
+  nugetDeps = ./deps.nix;
+  executables = [ executable ];
+  patches = [ ./targetframework.patch ];
+  runtimeDeps = [ fontconfig openal xorg.libX11 xorg.libICE xorg.libSM ];
+  src = fetchFromGitHub {
+    owner = "KnossosNET";
+    repo = "Knossos.NET";
+    rev = "v${version}";
+    hash = "sha256-nMxi6OPoyXrrjPa60DM7ZSCQ/zOuwWNuDm0rb7oO3Yc=";
+  };
+  meta = with lib; {
```

Comment:
meta.maintainers must be set

Reply:
Must have missed setting that while trying to figure out all the other things that had to be done correctly